header/p2p/peer_tracker.go

```diff
@@ -12,23 +13,58 @@ import (
 
 type peerTracker struct {
```

Comment:
The last suggestion is to logically sort all the fields and document what trackedPeers are